core/src/sample_performance_service.rs

```diff
@@ -75,18 +87,10 @@ impl SamplePerformanceService {
 
             if elapsed.as_secs() >= SAMPLE_INTERVAL {
                 now = Instant::now();
-                let (bank, highest_slot) = {
```

Comment:
We do actually need the slot for writing to blockstore on L102

Reply:
You are right.
I shared it a bit too quickly %)
Wanted to make sure you'll see it before you merge your PR.
Should be passing the CI now.